examples/research_projects/lxmert/utils.py

```diff
@@ -182,7 +182,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: str, **kwargs):
     def get_config_dict(cls, pretrained_model_name_or_path: str, **kwargs):
         cache_dir = kwargs.pop("cache_dir", None)
         force_download = kwargs.pop("force_download", False)
-        resume_download = kwargs.pop("resume_download", False)
+        resume_download = kwargs.pop("resume_download", None)
```

Comment:
I have been told that usually the `examples/` folder is not maintained or updated over time.